org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/util/SonarLintWebView.java

```diff
@@ -85,7 +90,8 @@ public SonarLintWebView(Composite parent, boolean useEditorFontSize) {
       browser = new Browser(this, SWT.FILL);
       addLinkListener(browser);
       browser.setLayoutData(new GridData(SWT.FILL, SWT.FILL, true, true, 1, 1));
-      browser.setJavascriptEnabled(false);
+      browser.setJavascriptEnabled(true);
```

Comment:
As discussed, I now think we should move away from this `BrowserFunction` + JavaScript solution. I tried with a simple addition to the `LocationListener`, and it works.

Reply:
Done.